modules/base/base/module.py

```diff
@@ -341,10 +349,14 @@ async def autothread_unset(self, ctx, channel: discord.TextChannel = None):
     async def on_message(self, message: discord.Message):
         if message.author.bot:
             return
-        if AutoThread.get(message.guild.id, message.channel.id) is None:
+        thread_settings = AutoThread.get(message.guild.id, message.channel.id)
+        if thread_settings is None:
```

Comment:
This won't ever happen, right?

This is the place where the potential downgrade of maximal duration should be applied: if the guild level is 1 and `thread_settings` wants to set  a week, downgrade it to the current maximum of three days.